src/app/components/Navbar.jsx

```diff
@@ -0,0 +1,98 @@
+"use client";
+import { useState } from "react";
+import Link from "next/link";
+import Image from "next/image";
+
+const links = [
+    { href: "/beta-reading", label: "Beta reading" },
+    { href: "/team", label: "Team" },
+    { href: "/mentorship", label: "Mentorship" },
+    { href: "/blog", label: "Blog" },
+    { href: "/workshops", label: "Workshops" },
+];
+
+const Navbar = () => {
+    const [menuOpen, setMenuOpen] = useState(false);
+
+    return (
+        <nav className="relative flex items-center bg-light shadow-lg w-full">
+            {/* Logo */}
+            <div className="bg-brand-dark flex items-center justify-center w-20 h-16 md:w-60 md:h-30">
```

Comment:
- Navbar image looks slightly pixelated even though the original logo is of high quality:

```suggestion
            <div className="bg-brand-dark relative w-20 h-16 md:w-60 md:h-30 flex items-center justify-center">
  <Image
    src="/logo.png"
    alt="Story Bridge Logo"
    fill
    className="object-contain"
    priority
  />
</div>
```
<img width="331" height="171" alt="image" src="https://github.com/user-attachments/assets/36c3d4a3-1b64-48c9-b68c-cff556c0c68d" />